stack.yaml

```diff
@@ -131,3 +142,11 @@ ghc-options:
   cardano-sl:            -Werror
   cardano-sl-tools:      -Werror
   cardano-sl-lwallet:    -Werror
+  cardano-sl-explorer:   -Werror
+
+# Added for explorer
+flags:
+  cardano-sl:
+    with-explorer: True
```

Comment:
If we do want to compile without _explorer_ and have it as a submodule, we need to remove the linker flags here - https://github.com/input-output-hk/cardano-sl/blob/6c8d85aea1dc4f980876668452130b38a3b2d5b3/cardano-sl.cabal#L450-L454 which would remove gold-linker optimizations from `cardano-sl`. The alternative? Switch the flag to true in `cardano-sl.cabal`, remove from `stack.yaml`.